pkg/v1/types/types.go

```diff
@@ -35,6 +35,7 @@ const (
 	DockerManifestSchema2       MediaType = "application/vnd.docker.distribution.manifest.v2+json"
 	DockerManifestList          MediaType = "application/vnd.docker.distribution.manifest.list.v2+json"
 	DockerLayer                 MediaType = "application/vnd.docker.image.rootfs.diff.tar.gzip"
+	DockerLayerZstd             MediaType = "application/vnd.docker.image.rootfs.diff.tar.zstd"
```

Comment:
Yes, to my surprise, such thing exists. See https://github.com/moby/buildkit/pull/3968

Reply:
One more prooflink: https://github.com/containerd/containerd/pull/9859

Reply:
🙈